src/Entity/Adherent.php

```diff
@@ -889,12 +889,18 @@ public function getMembershipFor(Committee $committee): ?CommitteeMembership
         return null;
     }
 
-    public function getCitizenProjectMemberships(): CitizenProjectMembershipCollection
+    public function getCitizenProjectMemberships($withoutRefused = false): CitizenProjectMembershipCollection
     {
         if (!$this->citizenProjectMemberships instanceof CitizenProjectMembershipCollection) {
             $this->citizenProjectMemberships = new CitizenProjectMembershipCollection($this->citizenProjectMemberships->toArray());
         }
 
+        if ($withoutRefused) {
+            $this->citizenProjectMemberships->filter(function (CitizenProjectMembership $membership) {
```

Comment:
1) `return` missing
2) maybe add `filterRefusedProject` method to `CitizenProjectMembershipCollection` class?